### ISSUE TITLE
how does 'img_guidance_scale' influence the image generation

### ISSUE THREAD

**Issue by zc1023** [VectorSpaceLab/OmniGen#124]
Hi there. I'm interested in your use of input images without text prompts as conditional input to achieve the CFG. Do you have any ablation studies or references to show it is useful?

**Comment by staoxiao** [comment]
The `img_guidance_scale ` is introduced in https://arxiv.org/pdf/2211.09800

**Comment by able2608** [comment]
Just for quick reference (for sake that the paper is too hard to follow): higher means the generated image will be closer to the original one, lower gives the model more freedom to edit the image. You'll need to tweak this value for each specific task and there are no magic number that fits all. However (specifically for this model) even values as low as 1.1 can give faithful results, so don't be afraid to lower this value if the desired edit cannot be done.
